src/main/java/org/opentripplanner/routing/framework/DebugTimingAggregator.java

```diff
@@ -1,237 +1,242 @@
 package org.opentripplanner.routing.framework;
 
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Metrics;
+import io.micrometer.core.instrument.Timer;
 import java.util.ArrayList;
 import java.util.List;
-import javax.annotation.Nullable;
 import org.opentripplanner.api.resource.DebugOutput;
 import org.opentripplanner.api.resource.TransitTimingOutput;
+import org.opentripplanner.ext.actuator.ActuatorAPI;
+import org.opentripplanner.util.OTPFeature;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Keeps account of timing information within the different parts of the routing process, and is
- * responsible of logging that information.
+ * responsible for logging that information.
  */
 public class DebugTimingAggregator {
-  private static final Logger LOG = LoggerFactory.getLogger(DebugTimingAggregator.class);
 
-  private long startedCalculating;
-  private long finishedPrecalculating;
+  private static final Logger LOG = LoggerFactory.getLogger(DebugTimingAggregator.class);
 
-  private long startedDirectStreetRouter;
-  private long finishedDirectStreetRouter;
+  private static final long nanosToMillis = 1000000;
+
+  public static final Clock clock = OTPFeature.ActuatorAPI.isOn()
+          ? ActuatorAPI.prometheusRegistry.config().clock()
+          : Clock.SYSTEM;
+
+  private static final MeterRegistry registry = OTPFeature.ActuatorAPI.isOn()
```

Comment:
I see this code in another place. Perhaps we can extract a method to get the correct `MeterRegistry` instance.

Reply:
`ActuatorAPI` should be updated to (correctly?) add the `prometheusRegistry` to the `Metrics.globalRegistry`:
```
        Metrics.globalRegistry.add(prometheusRegistry);
```

And then `Metrics.globalRegistry` may be used in all places.

Which would also allow moving the metrics initialization from `ActuatorAPI` into a common place.

Reply:
We can place it in a static context in `org.opentripplanner.util.logging`, maybe?